docs/immortal-empires/high-elves.md

```diff
@@ -0,0 +1,41 @@
+# High Elves
+
+## Tyrion - Eataine
+
+* **Route I - Defender of Ulthuan**
+  * Prince Tyrion is the Defender of Ulthuan and the protector of its people, a heavy burden entrusted to him by the Phoenix King Finubar. Among the Dark Elves of Naggaroth he is known as the Reaper, to the Goblins of Red-Axe Pass he is Orcbane, and to the north the Norse know him as Mankiller. For two centuries he has stood between the Elves of Ulthuan and their many foes as an unbreakable shield against the darkness.
+  * Destroy factions:
+    * Seducers of Slaanesh, The Dreadfleet, Broken Axe, The Drowned
+  * **Reward**:
+    * _The Best Defence is a Good Offence_: Income from Ports: +50%, +10 Leadership (factionwide) 
+_Reward flavour text:_ Not since the Age of Colonisation have the Elves known such prosperity.
+icon_ritual_currency_hef_bundle.png
+
+* **Route II - All's Fair in Love and War**
+  * Malekith's ruinous ambitions have brought death and destruction to Ulthuan for centuries. Elf has fought against elf for far too long. The civil war must end.
+  * Destroy factions: 
+    * Naggarond
+  * Tyrion concerns himself only with the twin duties of protecting Ulthuan and its Everqueen, the beloved Alarielle. However, the latter is seen less as a duty than it is a calling of the heart - it is common knowledge that Tyrion is Alarielle's consort, and has been for many years.
+  * Occupy settlement:
+    * Lothern, Gaean Vale
+  * Build Landmark:
+    * Court of the Phoenix King
+  * **Reward**: 
+    * _Dawn of a New Age_: All remaining Dark Elf factions suffer -10 control, Recruitment duration: -1 turn for all units (minimum 1), Recruit rank: +3 for all units
+_Reward flavour text:_ With the Witch King's demise, revolts have broken out across Naggaroth. Thousands of slaves and druchii defectors flee the Black Cities, and some even flock to your banner.
+dlc10_assassination_targets.png
+    * _The Everchild_: Influence per turn: +10, Corruption -2
+_Reward flavour text:_ Ostensibly the progeny of the Everqueen and the Phoenix King, some whisper that Aliathra's pedigree is... less than certain.
+dlc10_power_of_nature.png
+
+* **Route III - The Fate of the Soul**
+  * The fate of Tyrion's soul teeters on a blade-edge. Will he succumb to Aenarion's curse and salve his people's wounded pride with the blood of old enemies, or will he put aside these grievances and reforge old alliances that will stand together against the forces of Chaos?
+  * Occupy settlements:
+    * Shrine of Khaine, Karaz-a-Karak, The Oak of Ages
+  * **Reward**:
+    * _Reunification_: All Wood Elf Legendary Lords become vassals. +100% tribute from vassals.
+_Reward flavour text:_ After centuries of isolation, the Asrai now answer to the Phoenix Court.
+worldroots_health.png
+    * _The Original Phoenix Crown_: All Dwarf Legendary Lords become vassals. +100% Allegiance points gained for all races.
```

Comment:
👀👀👀 This reward is.... wow. The _idea_ is excellent, but isn't it too strong?

Reply:
Well, what are your concerns about its strength?

Are you worried about the income from vassals? In my experience, it's really not much.

Reply:
Not even specifically the income, just handing out multiple free vassals is an insane effort multiplier in your favour. Vassalising those factions the usual way would take multiple dozens of turns. If we're collectively happy with that scale of reward then I'm all for it, thematically it's fantastic.

Reply:
Yeah, vassalising is very difficult to achieve, but it doesn't actually give you much of an advantage once you do it. Only difference between it and military alliance is income.

Reply:
I do not like mixing both Wood Elves and Dwarves here. I would stick with the formers which feels already powerful enough. Instantly getting 20% of the map as your vassals just for the sake of why not feels game-breaking.

Reply:
Objective will be a little light with just one. Rather than doing a complete rewrite I will instead just nerf the Phoenix Crown payload to: Income from vassals: +50%, Income from trade: +10% for every ally.

Reply:
Fine for me. Just do not make the dwarfs your vassals 🍡